student.rb

```diff
@@ -3,13 +3,17 @@
 class Student < Person
   attr_accessor :classroom
 
-  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+  def initialize(age, name: 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
```

Comment:
- [ ] It would be nice if you don't remove the classroom from the initialize because you have just created a method with the name of the join_classroom which takes classroom as an input but in your case this method should work automatically and you won't need to use the join_classroom method manually to add classroom to a student 🤔 So kindly put the classroom back to make your app automated 👍